common/views/CommitmentListItem.jsx

```diff
@@ -0,0 +1,224 @@
+    primaryText: React.PropTypes.node,
+    secondaryText: React.PropTypes.node,
+    secondaryTextLines: React.PropTypes.oneOf([1, 2]),
+  },
+
+  getDefaultProps() {
+    return {
+      secondaryTextLines: 1,
+    };
+  },
+
+  getInitialState() {
+    return {
+      hovered: false,
+      isKeyboardFocused: false,
+      touch: false,
+    };
+  },
+
+  render() {
+    let {
+      leftIcon,
+      onKeyboardFocus,
+      onMouseLeave,
+      onMouseEnter,
+      onTouchStart,
+      primaryText,
+      secondaryText,
+      secondaryTextLines,
+      style,
+      ...other,
+    } = this.props;
+
+    let textColor = this.context.muiTheme.palette.textColor,
+        hoverColor = ColorManipulator.fade(textColor, 0.1),
+        singleAvatar = !secondaryText && leftIcon,
+        singleNoAvatar = !secondaryText && !leftIcon,
+        twoLine = secondaryText && secondaryTextLines === 1,
+        threeLine = secondaryText && secondaryTextLines > 1,
+
+    styles = {
+      root: {
+        backgroundColor: (this.state.isKeyboardFocused || this.state.hovered) &&
+          !this.state.rightIconButtonHovered &&
+          !this.state.rightIconButtonKeyboardFocused ? hoverColor : null,
+        color: textColor,
+        display: 'block',
+        fontSize: 16,
+        lineHeight: '16px',
+        position: 'relative',
+        transition: Transitions.easeOut(),
+        cursor: 'pointer',
+      },
+
+      innerDiv: {
+        paddingLeft: leftIcon ? 72 : 16,
+        paddingBottom: singleAvatar ? 20 : 16,
+        paddingTop: singleNoAvatar || threeLine ? 16 : 20,
+        position: 'relative',
+      },
+
+      icons: {
+        height: 24,
+        width: 24,
+        display: 'block',
+        position: 'absolute',
+        top: twoLine ? 12 : singleAvatar ? 4 : 0,
+        padding: 12,
+      },
+
+      leftIcon: {
+        color: Colors.grey600,
+        fill: Colors.grey600,
+        left: 4,
+      },
+
+      primaryText: {
+        margin: 0,
+      },
+
+      secondaryText: {
+        fontSize: 14,
+        lineHeight: threeLine ? '18px' : '16px',
+        height: threeLine ? 36 : 16,
+        margin: 0,
+        marginTop: 4,
+        color: Typography.textLightBlack,
+
+        //needed for 2 and 3 line ellipsis
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
+        whiteSpace: threeLine ? null : 'nowrap',
+        display: threeLine ? '-webkit-box' : null,
+        WebkitLineClamp: threeLine ? 2 : null,
+        WebkitBoxOrient: threeLine ? 'vertical' : null,
+      },
+    },
+
+        primaryTextIsAnElement = React.isValidElement(primaryText),
+        secondaryTextIsAnElement = React.isValidElement(secondaryText),
+
+        mergedRootStyles = this.mergeAndPrefix(styles.root, style),
+        mergedInnerDivStyles = styles.innerDiv,
+
+        mergedPrimaryTextStyles = primaryTextIsAnElement ?
+      this.mergeStyles(styles.primaryText, primaryText.props.style) : null,
```

Comment:
this line and L129 should indent more (at least more than the statement with `?`)